.npmignore

```diff
@@ -0,0 +1,3 @@
+test
```

Comment:
I've also added an npmignore file. Do we prefer to use the files property in the package.json?

Verified what files would be packaged with:

```bash
npm pack && tar -tf eslint-config-braintree-1.0.1.tgz
```